phpseclib/Crypt/Base.php

```diff
@@ -296,9 +296,9 @@ class Crypt_Base
     /**
      * Does internal cipher state need to be (re)initialized?
      *
-     * @see setKey()
-     * @see setIV()
-     * @see disableContinuousBuffer()
+     * @see Crypt_Base::setKey()
```

Comment:
static, to me, implies that the method needs to be called statically, which is not the case here.

I could do self but the 1.0 branch is supposed to be compatible with 4.x (with PHP_Compat). Of course I guess making changes to phpDoc comments doesn't really change the versioning requirements so I guess I can do it.

I guess phpDoc just kinda translates `self` to `Crypt_Base` (assuming that `self` occurs in `Crypt_Base`) when it's building the documentation?